client/src/components/geneExpression/quickGene.tsx

```diff
@@ -50,14 +51,27 @@ function QuickGene() {
     fetch();
 
     async function fetch() {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on commit
       if (annoMatrix !== (prevProps as any)?.annoMatrix) {
         const { schema } = annoMatrix;
         const varIndex = schema.annotations.var.index;
 
         setStatus("pending");
         try {
           const df: Dataframe = await annoMatrix.fetch("var", varIndex);
-          setStatus("success");
+          let dfIds: Dataframe;
+          const geneIdCol = "feature_id";
+
+          // if feature id column is available in var
+          if (annoMatrix.getMatrixColumns("var").includes(geneIdCol)) {
+            dfIds = await annoMatrix.fetch("var", geneIdCol);
+            console.log("id, success");
```

Comment:
left in debug code?

Reply:
I removed the console.log, but the rest I think is still important for datasets that contain feature_id columns